trio/_core/_tests/test_util.py

```diff
@@ -1 +1 @@
-import pytest
+import pytest  # noqa: F401
```

Comment:
Hm, why is this staying?

Reply:
I assumed that importing pytest was just what the test *did* (for some strange reason), but looking at the history of the file it seems like it should probably have been removed a long time ago.